### PR TITLE
EL-1019 add new hints for all levels of help

### PR DESCRIPTION
**Pull request by willc-work**
[Jira ticket](https://dsdmoj.atlassian.net/browse/EL-1019)

## What changed and why

new hint text on client and partner assets screens

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing
- Branch is generally up to date with main Github - definitely no conflicts
- No unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- PR description says *what* changed and *why*, with a link to the JIRA story.
- Diff has been checked for unexpected changes being included.
- Commit messages say why the change was made.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1141,7 +1141,12 @@ en:
       bank_accounts:
         label: Money in bank accounts
         hints:
-          - For example, money in current or savings accounts, ISAs, or child accounts your client is a signatory on. Do not include any government Cost of Living Payments owed to your client.
+          - For example, money in current or savings accounts, ISAs, or child accounts your client is a signatory on.
+          - Do not include any government Cost of Living Payments owed to your client.
+          - Enter 0 if this does not apply.
+        with_partner_hints:
+          - For example, money in current or savings accounts, ISAs, or child accounts your client is a signatory on. If the client and partner share any joint bank accounts, include these.
```
Comment (MazOneTwoOne) [inline config/locales/en.yml:1148]: `If the client and partner share any joint bank accounts, include these.` This should be on line 1144 as part of the client hint

File: config/locales/en.yml [hunk 2]
```diff
@@ -1151,9 +1156,20 @@ en:
         hints:
           - For example, shares in a public limited company, unit trusts or capital bonds.
           - Enter 0 if this does not apply.
+        with_partner_hints:
+          - For example, shares in a public limited company, unit trusts or capital bonds. Include any jointly owned by the client and partner.
+          - Enter 0 if this does not apply.
```
[the remark this answers is not in the document]
Comment (patrick-laa) [reply config/locales/en.yml:1161]: Hold up - 1157 is what gets shown on the partner screen or on the client screen _if the client does not have a partner_.  1160 is what gets show on the client screen _if the client has a partner_. So I think this is as it should be.

File: config/locales/en.yml [hunk 3]
```diff
@@ -1151,9 +1156,20 @@ en:
...
         input: Total of all investments
       valuables:
         label: Valuable items worth £500 or more
+        certificated_with_partner_hints:
+          - If you have more than one valuation for any of these valuable items, use the highest. Keep a record of where the valuation is from for evidence, as it may be checked if you make an application for legal aid.
+          - Include any jointly owned by the client and partner.
```
Comment (MazOneTwoOne) [inline config/locales/en.yml:1167]: `Include any jointly owned by the client and partner.` this should be with client hint between 1174-1175

File: config/locales/en.yml [hunk 4]
```diff
@@ -1151,9 +1156,20 @@ en:
...
+          - Enter 0 if this does not apply.
+        controlled_with_partner_hints:
+          - If you have more than one valuation for any of these valuable items, use the highest. Keep a record of where the valuation is from for evidence, as it may be checked at file audit or assessment stage.
+          - Include any jointly owned by the client and partner.
```
Comment (MazOneTwoOne) [inline config/locales/en.yml:1171]: `Include any jointly owned by the client and partner.` this should be with client between 1177-1178

File: config/locales/en.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (MazOneTwoOne) [reply config/locales/en.yml:1161]: Ignore me - these were meant to be draft comments as part of main review (but I obviously don't know how to use GitHub to code review). 

Yes you are right it is meant to be like this - was reading the ticket wrong. 😳

**Review by MazOneTwoOne — APPROVED** [review]
This looks good and also have noted the wording mis-match on the 'certificated valuables' question. Happy to approve.

**Pull request merged**